docs/usage/getting-started/migrating-secrets.md

```diff
@@ -0,0 +1,93 @@
+# Migrating Secrets from Repo Config to App Settings
+
+Storing of encrypted secrets in the repo is not supported starting from 01-Sep-2024 for Mend Renovate Cloud apps.
```

Comment:
I'm not sure this date is realistic if we haven't yet begun to warn users

Reply:
I'll bump it to 1-Oct-2024